broadinstitute/single_cell_portal_core
# Remove old wiz & flag (SCP-4076)

Remove the old upload wizard code and the feature flag

To Test:
- Pull this branch
- Run the migration script via `bin/rails db:migrate`
- Go to the Feature Flags page and see that the react upload wizard flag is no longer there
- Go to the upload wizard and confirm all works.
![Screen Shot 2022-02-17 at 11 22 36 AM](https://user-images.githubusercontent.com/54322292/154524702-97e387ef-ef74-4b19-845c-c3aa12d074d7.png)
